.github/workflows/i18n_update.yaml

```diff
@@ -24,20 +24,6 @@ jobs:
         uses: actions/checkout@v3
         with:
           submodules: 'true'
-      - name: Checkout mozilla-vpn-client-l10n Git
-        run: |
-          git submodule update --init --remote src/apps/vpn/translations/i18n/
-      - uses: peter-evans/create-pull-request@v5
-        id: cpr
-        with:
-          # This may not be github, cuz gh actions cannot self approve
-          author: GitHub <noreply@github.com>
-          committer: GitHub <noreply@github.com>
-          commit-message: "[Bot] Update i18n"
-          branch: i18n_automation
-          delete-branch: true
-          title: "[Bot] Update i18n"
-          token: ${{ secrets.WIKI_TOKEN }}
```

Comment:
This need's to say, this is the bot that will daily try to update the languages :)